### DISCUSSION TITLE
Error 8992 - Integrity check failed for master database in Contained Availability Group

### DISCUSSION THREAD

**Discussion by vdusan** [olahallengren/sql-server-maintenance-solution#817] [Questions & Answers]
Integrity check failed (Error 8992) for master database in Contained Availability Group. I have a job for user databases integrity check but it fails because it checks master database in Contained Availability Group also. Does anybody know how to configure integrity jobs for all databases in Contained Availability Group (including master).

**Comment by olahallengren** [comment]
This looks like a bug in SQL Server or a real corruption issue.

My script is executing the SQL Server DBCC CHECKDB command.

You don't need to do anything special to perform integrity checks of contained availability groups. You can have two jobs, one for USER_DATABASES and one for SYSTEM_DATABASES, both jobs running against the SQL Server instance. The system databases for the contained availability group will be checked as part of USER_DATABASES.

I have tested this on SQL Server 2022 CU25 and SQL Server 2025 CU4 some time ago (the latest CUs at the time).

Could you try installing the latest cumulative update package?
